owl/SOMA-ACT.owl

```diff
@@ -1346,6 +1376,23 @@ As such, an explicit assertion is needed to make a Situation a nonmanifested one
     
 
 
+    <!-- 
+    ///////////////////////////////////////////////////////////////////////////////////////
+    //
+    // Annotations
+    //
+    ///////////////////////////////////////////////////////////////////////////////////////
+     -->
+
+    <rdf:Description rdf:about="http://www.ease-crc.org/ont/SOMA.owl#transitionsBack">
+        <rdfs:label xml:lang="en">transitions back</rdfs:label>
```

Comment:
this seems odd. I assume the properties are defined somewhere else? the label should be moved there